src/app/utils/fundedStatus.js

```diff
@@ -0,0 +1,8 @@
+const fundedStatus = [
```

Comment:
we have model/thema.ts now, lets use that

Reply:
https://github.com/Microsoft/TypeScript/issues/17198

Here's an example, please let us know if you have any questions @pedro-surf